src/presentation/api/volunteer.ts

```diff
@@ -223,4 +225,47 @@ export class VolunteerAPI extends Controller {
         })
       );
   }
+
+  /**
+   * Post volunteer hours
+   *
+   */
+
+  @Post('hours')
+  // @Security('jwt', ['bookPermission'])
+  @SuccessResponse(201, 'Successfully posting hours')
+  // @Response<VolunteerError>(400, 'Could not find volunteer') // como colocar o erro e qual status ?
```

Comment:
Tsoa does not support multiple errors for the same error code. There's a work around as described here: 
https://stackoverflow.com/questions/69377605/tsoa-multiple-responses-for-same-status-code

Another option could be to use status `409`(Conflict) for hours already registered in the month.